### DISCUSSION TITLE
Cura 5.4

### DISCUSSION THREAD

**Discussion by Fropzs** [phnxdrgn536/FLSun-v400#38] [Q&A]
Does the profile works on cura 5.4?

Use them on 5.2 and works flowless👍🏼

But installed 5.4 last night and tryed the same profile and same print as 5.2, the nozzle hitted my support and crash the hole print
on layer 5-6,

Did nothing to the new 5.4, just installed, the profile was transfer from 5.2 or do i need to transer it manual ?

**Comment by phnxdrgn536** [comment]
It should just be installing the profile in the 5.4 folders

**Comment by ac1dl0rd** [comment]
Hi,
I tried several things but Cura always crashes after a restart of the software. Corrupt file etc?

**Comment by phnxdrgn536** [comment]
Do you have the crash report so I can look and see what’s happening?

**Reply by ac1dl0rd** [reply]
I started fresh, I believe the problem is that now V400 is default in Cura configuration so there are .json files in C:\Program Files\UltiMaker Cura 5.3.0\share\cura\resources\definitions

it seems to ignore the one in appdata

I replaced all the files in the Share\cura by yours for the json etc.

**Comment by phnxdrgn536** [comment]
Ahhhh ok I think I know what’s wrong… since they added the V400 in cura 5.4
I’ll have to rework the profiles to work with the default V400 file. I’ll
start working on that. For now it looks like my profiles are limited to 5.3
and below

**Reply by ac1dl0rd** [reply]
even with 5.3 the problem was present

THANKS :)

**Reply by FinboySlick** [reply]
I suspect this could be the same #37  I reported earlier.  Glad to see I wasn't alone if this is the case.
